### PR TITLE
Kasa library to pypi

### LINKED ISSUE #8: Integrate BERT models into Kasa lib

### PR DESCRIPTION
**Pull request by hotlynnn**
The processing.py file has been exported to the main pypi.org repo and officially working. Anyone can pip install kasa now to use the library.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,13 @@
...
     description='A library of translation and other NLP tools for the Ghanaian language Twi',
+    author='Dr.Paul Azunre',
+    author_email='azunre@gmail.com',
+    license='MIT',
     packages=['Kasa'],
-    install_requires=['gensim >= 3.8.1'],
-    include_package_data=True,
+    install_requires=['gensim == 3.8.1'],
+    include_package_data=True, 
+    include_package_data=True,
```
Comment (joelbudu) [inline setup.py]: ```include_package_data=True``` should appear only once I believe

Comment (hotlynnn) [reply setup.py]: yeah, I think it was a typo
Let me rectify it

Comment (joelbudu) [reply setup.py]: The HISTORY.md and LICENSE files shouldn't be moved. Should remain in the base directory

Comment (hotlynnn) [reply setup.py]: Alright
I will do that...Anymore?

Comment (joelbudu) [reply setup.py]: no that's all.. you may just update the PR

Comment (hotlynnn) [reply setup.py]: Alright

Comment (MarkMarcel) [reply setup.py]: pip install kasa works without any issues

**Review by MarkMarcel — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (hotlynnn) [reply setup.py]: Joel raised some issues so I will rectify them and update the PR

**Pull request merged**